src/utils/tutorials.js

```diff
@@ -0,0 +1,10 @@
+import tutorialsList from '../static/tutorials.json'
+
+export const getCurrentTutorial = route => (
+  route.props && route.props.default && route.props.default.tutorialId &&
```

Comment:
If vue-cli 3 already supported optional chaining we could have this a bit prettier 😄  but i'm pretty sure it doesn't.

Reply:
Yeah we would need to add the babel plugin manually.
Another solution would be to add `lodash` and use `lodash.get`. Since it is already in the dependency tree, no bundle size issues there.